name,value,relevant log,explanation
io.bytes.per.checksum,819562056,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.java.lang.NegativeArraySizeException: -1213876088
	",property-item-hit	io
hadoop.security.groups.cache.background.reload.threads,3,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.java.lang.NegativeArraySizeException: -1213876088
	",value-hit
dfs.namenode.maintenance.replication.min,1,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.java.lang.NegativeArraySizeException: -1213876088
	",value-hit
dfs.client.read.shortcircuit.streams.cache.size,256,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.java.lang.NegativeArraySizeException: -1213876088
	",property-item-hit	read
